source/migration/JetifierWrapper/build.gradle

```diff
@@ -35,4 +35,5 @@ compileJava {
 
 jar {
     from { configurations.compile.collect { it.isDirectory() ? it : zipTree(it) } }
+    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
```

Comment:
The `jar` task is merging all the dependencies, and some files are duplicated because they exist in multiple dependencies. 
This is "safe" because we always start with our jar file, and then merge the dependencies in. Since there is no actual class duplicates, these are just manifests and readmes that we don't care about.